castanet/counter.py

```diff
@@ -43,7 +43,14 @@ def count_whileloops(cast_dict):
 
 
 def count_forloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of while loops in a Python program.
+
+    Args:
+        cast_dict: counts the number of for loops in a file
```

Comment:
The cast_dict argument does not count the number of for loops in a file, rather it is a dictionary of files and their corresponding CAST's. This should be replaced with a statement like that.